### ISSUE TITLE
Is there merit to making audioloader a Volumio/Moode plugin?

### ISSUE THREAD

**Issue by ibex-are-goats** [krisek/audioloader#3]
First off, great work!

I don't like the Volumio/Moode interfaces. Behind the times compared to yours. I suspect this interface can run on an Rpi. Is there merit to making audioloader easily installable on those distros, perhaps like a Volumio plugin? Thanks for your answer :)

**Comment by krisek** [comment]
No problem, you are right, Audioloader feels very well on a Pi. 

I don't have too much experience with Volumeio, actually I'm using the various ancestors of Audioloader for ages now, there are several reasons for this, the most important is the dash: I like to listen albums, I have a lot of them, so I need to have my current favourites / my recents and some random selection at hand. None of the music services/organizer/software I tried could give me this functionality. Audioloader is stil very specialized for this use case (ie listening to albums), hence there are a lot of features not there what you can see in other solutions. I doubt if any integration would make sense into Volumeio/Mopidy/etc. without completely blowing up Audioloader. Though it is a free software, so anybody can do it, until he/she follows the rules.

For your other question about packaging: I was thinking about this, but if I look at the various options like packaging into deb/rpm or snap/flatpak or as a Python package all have some drawbacks and I really couldn't decide. Finally, this is a web application, web applications have their own update mechanisms which I never wanted to write, so I falled back to this low-budget option. For the time being it is okay, though updates are already headache. (For example: I'm almost ready with the UPnP functionality and I have no clue how people who already installed will get it... they should regularly pull, I don't expect this will happen). Any help/ideas are appreciated here!

Long story short: if you don't mind I close the issue: IMHO there is no merit in packaging it as a Volumio plugin, and for sure I won't do it in the near future. 

Feel free to fork it though, and maybe we can integrate it into upstream.

**Comment by ibex-are-goats** [comment]
Thank you for the reply, I'm very glad to just inspire some thinking!! :) :+1:
